src/polygonParts/controllers/polygonPartsController.ts

```diff
@@ -52,6 +54,14 @@ export type AggregationLayerMetadataHandler = RequestHandler<
   EntitiesMetadata
 >;
 
+export type ValidatePolygonPartsHandler = RequestHandler<
+  undefined,
+  ValidatePolygonPartsResponseBody,
+  ValidatePolygonPartsRequestBody,
+  undefined,
+  EntitiesMetadata
```

Comment:
why `EntitiesMetadata` as ReqQuery ?

Reply:
it is not, it is in req.locals (like in all other functions that get here through the transformerController